.github/workflows/continous-integration.yml

```diff
@@ -104,6 +104,7 @@ jobs:
     - name: Test Docs 📃
       run: |
         make install-full
+        poetry run python -c "from scripts import release; release.generate_changelog('major.minor.patch')"
```

Comment:
interesting solution 👍 I would have called towncrier directly, but this makes sense since it's closer to the release process